### PR TITLE
Database performance

### PR DESCRIPTION
**Pull request by Chimwemwe-127001**
- `SELECT COUNT(*) FROM visits where animal_id = 4;`
  **Before**
   visits-animal-id
     **After**
after-visits-animal-id

- `SELECT * FROM visits where vet_id = 2;`
  **Before**
   visits-vets-id
     **After**
after-visits-vets-id

- `SELECT * FROM owners where email = 'owner_18327@mail.com';`
  **Before**
   owners-email
     **After**
after-owners-email

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -24,4 +24,130 @@ SELECT neutered , MAX(escape_attempts) FROM animals GROUP BY neutered;
 -- What is the minimum and maximum weight of each type of animal?
 SELECT species, MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '6/1/1990' AND '12,31,2000' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '6/1/1990' AND '12,31,2000' GROUP BY species;
+
```
Comment (SajjadAhmad14) [inline queries.sql:28]: Please add indexes to foreign keys to increase the queries performance.

File: queries.sql [hunk 2]
```diff
@@ -24,4 +24,130 @@ SELECT neutered , MAX(escape_attempts) FROM animals GROUP BY neutered;
...
+
```
Comment (SajjadAhmad14) [inline queries.sql:29]: Please add pair of screenshots in the PR description with the results of the  `EXPLAIN ANALYZE` command for each of the above queries (in total it should be 6 screenshots).
The first screenshot should show how much time the query takes BEFORE adding indexes on foreign keys.
The second screenshot should show how much time the query takes AFTER the improvements introduced

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 2 above)
Comment (nevisende) [reply queries.sql:29]: Please check my codes I have already added Foreign keys so we haven't anythings to add. And If ypu check the screenshots you can see our execution time is really good

Comment (SajjadAhmad14) [reply queries.sql:29]: Hi, so the recommended solution is to add `Indexes` on foreign keys. Moreover, please add two screenshots for each query so they should be 6 in total.

**Review by AshakaE — APPROVED** [review]
### Hi @Chimwemwe-127001 & @SajjadAhmad14   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**